### PR TITLE
Scale endpoint and scale web hook

### PR DESCRIPTION
**Pull request by pchico83**

### CODE REVIEW

**Comment by pchico83** [comment]
updated

**Comment by pchico83** [comment]
updated

File: source/includes/webhooks.md [hunk 1]
```diff
@@ -21,16 +22,17 @@ Attribute | Description
...
 
-<!--
 ### Operations
 
 Operation | Description
 --------- | -----------
-Redeploy | Performs a `redeploy` service operation: every container with `STOPPED` or `RUNNING` state will be redeployed.
--->
+REDEPLOY | Performs a `redeploy` service operation.
+SCALE | Performs a `scale` service operation.
```
Comment (fermayo) [inline source/includes/webhooks.md]: shouldn't this be `SCALEUP`?

Comment (pchico83) [inline source/includes/webhooks.md]: updated, I am updating the code too now

**Pull request closed without merging**